### PR TITLE
chore(android): migrate DataSourceUtil to Kotlin

### PR DESCRIPTION
**Pull request by seyedmostafahasani**
## Summary
migrate DataSourceUtil to Kotlin

### Motivation
Migrate the codebase from Java to Kotlin

## Test plan

### CODE REVIEW

**Review by KrzysztofMoch — APPROVED** [review]

File: android/src/main/java/com/brentvatne/exoplayer/DataSourceUtil.kt
Comment (YangJonghun) [inline android/src/main/java/com/brentvatne/exoplayer/DataSourceUtil.kt:1]: I suggest to use object and @JvmStatic

Comment (seyedmostafahasani) [reply android/src/main/java/com/brentvatne/exoplayer/DataSourceUtil.kt:1]: I agree with your suggestion.

**Comment by freeboub** [comment]
There is a lintter issue on this PR !

**Comment by seyedmostafahasani** [comment]
@freeboub it's fixed!

**Review by YangJonghun — APPROVED** [review]

**Review by freeboub — APPROVED** [review]

**Review by freeboub — APPROVED** [review]

**Pull request merged**